examples/E2E-RAG-App/01_ingest_files.py

```diff
@@ -0,0 +1,181 @@
+import argparse
```

Comment:
I think we should simply call this directory `DocQA` with the organization:

- DocQA
  - app.py
  - README.md
  - scripts/
  - data/

Could we also avoid prefixing files `01_` or `02_` etc.

Reply:
fixed